### DISCUSSION TITLE
Arc Browser

### DISCUSSION THREAD

**Discussion by turtle-key** [catppuccin/catppuccin#2514] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

[www.arc.net/](https://arc.net/)

### Any additional comments?

It is a chromium based browser and an override to the official theme system would be appreciated 💪.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
This isn't possible as far as I know? Are you aware of any prior work on providing a preset theme like you are proposing for Arc?

**Reply by turtle-key** [reply]
Arc is relatively new on the market. It's chromium based, so I tried installing the Catppuccin Chrome theme, but it couldn't override the default theme system.

**Reply by uncenter** [reply]
Yes, Arc is a Chromium browser under the hood but the majority of the UI is in Swift and cannot be themed by Chrome theme extensions. You can use their basic color picker but you can't input specific colors - let alone import a preset color scheme like Catppuccin.

**Reply by turtle-key** [reply]
Thx. But it's still fckd up that i can't override it's theme or at least add non-default colors to it.☹️
Switching back to chromium or firefox(at least it's open source).

**Reply by minding-myown-business** [reply]
Arc Boosts can change the look of specific web pages on Arc, or (I _think, maybe_ ) change the look and feel of the whole browser. https://arc.net/boosts

**Reply by uncenter** [reply]
> Arc Boosts can change the look of specific web pages on Arc, or (I _think, maybe_ ) change the look and feel of the whole browser. https://arc.net/boosts

Boosts cannot change anything outside of the webpage itself, so no that won't work.

**Comment by sgoudham** [comment]
Going to close this as not planned give the thread above, if the theming situation changes in the future then we can re-open it.
